content/Unreal Tournament/Mutators/D/9/8/fddb01/double-enforcer-arena_[98fddb01].yml

```diff
@@ -0,0 +1,34 @@
+--- !<MUTATOR>
+contentType: "MUTATOR"
+firstIndex: "2022-05-21 20:43"
+game: "Unreal Tournament"
+name: "Double Enforcer Arena"
+author: "Unknown"
+description: "Players start with double enforcers."
+releaseDate: "2022-05"
```

Comment:
Release date: 1999-09-27 (for the UT Demo)
Author: Ob1-Kenobi
Source: http://www.ut-files.com/index.php?dir=Mods/&file=denforcerarena.rar